features/enable_fips_container.feature

```diff
@@ -0,0 +1,97 @@
+
+@uses.config.contract_token
+Feature: FIPS enablement in lxd containers
+
+    @series.xenial
+    @series.bionic
+    @series.focal
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: Attached enable of FIPS in an ubuntu lxd container
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        And I run `DEBIAN_FRONTEND=noninteractive apt-get install -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" -y openssh-client openssh-server strongswan openssl <libssl> libgcrypt20` with sudo, retrying exit [100]
+        And I run `ua enable fips<updates>` `with sudo` and stdin `y`
+        Then stdout matches regexp:
+            """
+            Warning: Enabling <fips-name> in a container.
+                     This will install the FIPS packages but not the kernel.
+                     This container must run on a host with <fips-name> enabled to be
+                     compliant.
+                     Please run `apt upgrade` to ensure all FIPS packages are updated to
+                     the correct version.
+            Warning: This action can take some time and cannot be undone.
```

Comment:
When answering no to this prompt I got a an unexpected error prompt.
I saw that on a bionic container after running the `fips-updates` test:

```
One moment, checking your subscription first
Warning: Enabling FIPS in a container.
         This will install the FIPS packages but not the kernel.
         This container must run on a host with FIPS enabled to be
         compliant.
         Please run `apt upgrade` to ensure all FIPS packages are updated to
         the correct version.
Warning: This action can take some time and cannot be undone.
Are you sure? (y/N) N
Unexpected error(s) occurred.
For more details, see the log: /var/log/ubuntu-advantage.log
To file a bug run: ubuntu-bug ubuntu-advantage-tools
```

Reply:
Oh weird, I thought I tested that but I guess it was early on implementing this and some things have changed. I'll look into it